### PR TITLE
Update Linux Qt + other Docker enhancements

### PR DESCRIPTION
**Pull request by slipher**
Now all 3 OSes are building on Qt 5.14.

### CODE REVIEW

**Comment by illwieckz** [comment]
Do the fact we would build Qt ourselves would allow us to build a more lightweight Qt (by not building useless stuff) because the final binary size is usually big.

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Before building the updater itself you need to build aria2 library
 ```
 cd aria2
 autoreconf -i
-./configure --without-libxml2 --without-libexpat --without-sqlite3 --enable-libaria2 --without-zlib --without-libcares --enable-static=yes ARIA2_STATIC=yes --without-libssh2 --disable-websocket --disable-nls
+./configure --without-libxml2 --without-libexpat --without-sqlite3 --enable-libaria2 --without-zlib --without-libcares --enable-static=yes ARIA2_STATIC=yes --without-libssh2 --disable-websocket --disable-nls --with-openssl
```
Comment (DolceTriade) [inline README.md:16]: What's our strategy for this? Are we statically compiling OpenSSL or dlopen'ing it?

**Comment by slipher** [comment]
> Do the fact we would build Qt ourselves would allow us to build a more lightweight Qt (by not building useless stuff) because the final binary size is usually big.

I've tried to minimize the amount of Qt stuff to build (see the MODULES stuff). I don't think this makes the actual binary smaller though; it just makes it faster/smaller to build Qt.

File: README.md (hunk 1 above)
Comment (slipher) [reply README.md:16]: It's static. You can check the commit message - I diffed the outputs of `ldd` to make sure we aren't adding unwanted dynamic dependencies.

Comment (DolceTriade) [reply README.md:16]: aight. FYI, 

`objdump -x <file> | grep NEEDED` is probably nicer than ldd. ldd includes transitive dependencies whereas objdump -x only shows the directly needed deps.

**Review by DolceTriade — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (slipher) [reply README.md:16]: Interesting, I tried this and it showed one additional (in the diff) added dependency `libXext.so.6`.

**Comment by DolceTriade** [comment]
> > Do the fact we would build Qt ourselves would allow us to build a more lightweight Qt (by not building useless stuff) because the final binary size is usually big.
> 
> I've tried to minimize the amount of Qt stuff to build (see the MODULES stuff). I don't think this makes the actual binary smaller though; it just makes it faster/smaller to build Qt.

ya, strip will generally remove all the unnecessary stuff anyways.

**Pull request merged**